blogs/pyladies-soflo.json

```diff
@@ -0,0 +1,20 @@
+{
+  "title": "PyLadies SoFlo YouTube channel",
+  "type": "YouTube channel",
+  "url": "https://www.youtube.com/channel/UCUPLdokEtQlQmbaW9UkJEVQ",
+  "photo_url": "https://yt3.googleusercontent.com/d0Q0c1jMSCBrfIGhWgs25pBWmym2UPZO3ex5NiAK6yhwM71AUDfyyAmeoe4xh5-sGuHRz800pw=s176-c-k-c0x00ffffff-no-rj",
+  "description": "PyLadies SoFlo YouTube channel. PyLadies SoFlo is a mentoring group with a focus on nurturing gender diversity in the South Florida Python community.",
+  "language": "en",
+  "rss_feed_youtube": "https://www.youtube.com/feeds/videos.xml?channel_id=UCUPLdokEtQlQmbaW9UkJEVQ",
+  "authors": [ 
+    {
+      "name": "Inessa Pawson"
+      "social_media": [{
+         "twitter": "inessapawson",
+         "github": "inessapawson",
```

Comment:
I believe the correct GitHub handle is `InessaPawson`, so it should be `"github": "InessaPawson"`